dags/sustainment/information_url_checker.py

```diff
@@ -74,20 +78,29 @@ def information_url_checker(**kwargs):
                     )
                     if 300 <= status_code < 400:
                         result_info[package_name] = (
-                            "Redirected URL:  " + information_url
+                            "Redirected URL:  "
+                            + str(status_code)
+                            + " "
+                            + information_url
                         )
                     elif 400 <= status_code < 500:
                         if status_code == 404:
                             result_info[package_name] = (
-                                "Page Not Found:  " + information_url
+                                "Page Not Found: "
+                                + str(status_code)
+                                + " "
+                                + information_url
                             )
-                        else:
+                        elif call.split("://")[1] not in white_list:
                             result_info[package_name] = (
-                                "Client Error:  " + information_url
+                                "Client Error: "
+                                + str(status_code)
+                                + " "
+                                + information_url
                             )
                     elif status_code >= 500:
-                        result_info[package_name] = "Server Error:  "
-                        +information_url
+                        result_info[package_name] = "Server Error: "
+                        +str(status_code) + "" + information_url
```

Comment:
Why is ` + "" +` there?

Reply:
good catch! this should be a space between the status code and information_url